torch/CMakeLists.txt

```diff
@@ -440,6 +440,7 @@ add_custom_command(
       --is-debug=${TORCH_VERSION_DEBUG}
       --cuda-version=${CUDA_VERSION}
       --hip-version=${HIP_VERSION}
+      --xpu-version=%{USE_XPU}
```

Comment:
@guangyey   any comments?

Reply:
I think we don't need this at this stage. PyTorch uses `torch.version.hip` to check if the current cuda is masqueraded by hip because they share the almost same code. It is not ambiguous to XPU.